packages/appstore/src/components/modals/verification-docs-list-modal/verification-docs-list-modal.tsx

```diff
@@ -29,15 +59,18 @@ const ListItem = observer(({ id, text, status, route }: TListItemProps) => {
 
     return (
         <div className='verification-docs-list-modal__content-list-item' onClick={onClickItem}>
-            <Text size={isMobile ? 'xxs' : 'xs'} line_height='xl'>
-                <Localize i18n_default_text={text} />
-            </Text>
+            <div>
+                <Text size={isMobile ? 'xxs' : 'xs'} line_height='xl'>
+                    <Localize i18n_default_text={text} />
+                </Text>
+            </div>
             {status === AUTH_STATUS_CODES.NONE || (id === 'tax' && status === 0) ? (
                 <LabelPairedChevronRightCaptionBoldIcon />
             ) : (
-                <Text size={isMobile ? 'xxs' : 'xs'} line_height='xl'>
-                    <Localize i18n_default_text={status} />
-                </Text>
+                <div className='verification-docs-list-modal__card'>
+                    <StatusBadge account_status={status} icon={badge_icon} text={badge_text} icon_size={badge_size} />
+                    <LabelPairedChevronRightCaptionBoldIcon fill='#c7c7c7' />
```

Comment:
just to clarify, is this color on design? why can't we use some of existing grey colors and pass here it as a variable?